src/tools/curve_format_main.cpp

```diff
@@ -177,12 +306,52 @@ int AllocateFiles(AllocateStruct* allocatestruct) {
 
 // TODO(tongguangxun) :添加单元测试
 int main(int argc, char** argv) {
+#ifdef WITH_SPDK
+    FILE *dpdk_log_stream = NULL;
+    cookie_io_functions_t io_funcs;
+    memset(&io_funcs, 0, sizeof(io_funcs));
+#endif
+
     google::ParseCommandLineFlags(&argc, &argv, false);
     google::InitGoogleLogging(argv[0]);
 
+#ifdef WITH_SPDK
+    io_funcs.write = glog_dpdk_log_func;
```

Comment:
What is the difference between the format of spdk and the format of ext4？

Reply:
It is same as ext4